recipe/meta.yaml

```
@@ -15,83 +18,182 @@ source:
     - pyproject.patch
 
 build:
-  number: 0
-  skip: true  # [aarch64]
+  number: 1
+  skip: True  # [aarch64]
   missing_dso_whitelist:
     - '*/RDKit*dll'  # [win]
 
 requirements:
+  # See https://conda-forge.org/docs/maintainer/knowledge_base/#placing-requirements-in-build-or-host and
+  # https://conda-forge.org/docs/maintainer/knowledge_base/#details-about-cross-compiled-python-packages.
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
+    - {{ stdlib('c') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - postgresql                          # [build_platform != target_platform]
+    - python                              # [build_platform != target_platform]
     - cmake
-    - jom  # [win]
+    - jom   # [win]
     - make  # [unix]
     - pkg-config
   host:
-    - libboost-devel
-    - libboost-python-devel
     - cairo
     - eigen
     - freetype
-    - python
+    - libboost-devel ={{ boost_version }}
+    - libboost-python-devel ={{ boost_version }}
```

Following comment above, this needs to be reverted:

```suggestion
    - libboost-devel
    - libboost-python-devel
```

And same elsewhere.

---

Done